pipeline/utils.groovy

```diff
@@ -143,6 +143,81 @@ def copy_public_keys() {
   }
 }
 
+/*
```

Comment:
parse comment message to find mentions for operator / controller PR number

set environment variables based on the comment